facets.json

```diff
@@ -4300,22 +4316,6 @@
       "@language" : "en",
       "@value" : "Yorkshire and The Humber"
     } ]
-  }, {
```

Comment:
Slightly worried that this has moved up to the top of the file, but I'm happy to ascribe to the change of Id. 🫣